test/gui/shared/steps/account_context.py

```diff
@@ -29,6 +29,8 @@ def step(context):
 
 @Then('the account with displayname "|any|" and host "|any|" should be displayed')
 def step(context, displayname, _):
+    if get_config('predefined_users'):
+        displayname = get_displayname_for_user(displayname.split()[0].strip())
```

Comment:
let's add a comment here
```suggestion
    if get_config('predefined_users'):
        # get first name from the display name
        displayname = get_displayname_for_user(displayname.split()[0].strip())
```